SUASSystem/SUASSystem/sda_converter.py

```diff
@@ -65,6 +66,17 @@ def set_waypoint(self, new_waypoint):
         self.obstacle_map.reset_waypoints()
         self.obstacle_map.add_waypoint(converted_waypoint)
 
+    def add_waypoint(self, new_waypoint):
```

Comment:
Do we really want an add waypoint method? I have the current functionality in place to simplify the calculations and the program logic to only operate with one waypoint. Do you think it is better to have the ability to have multiple waypoints?